src/utils/chain.ts

```diff
@@ -196,6 +196,30 @@ export const klimaAsset: IAssetWithBalance = {
   balanceWorthUsd: null,
 }
 
+export const plrDaoAsset: IAssetWithBalance = {
```

Comment:
please move these to different utils than `chain` maybe create another one called assets? these are not chain related